src/test/resources/pacts/publicapi-direct-debit-connector-search-by-mandate-three-results.json

```diff
@@ -0,0 +1,269 @@
+{
+  "provider": {
+    "name": "direct-debit-connector"
+  },
+  "consumer": {
+    "name": "publicapi"
+  },
+  "interactions": [
+    {
+      "description": "search transactions",
+      "providerStates": [
+        {
+          "name": "three transaction records exist",
+          "params": {
+            "gateway_account_id": "2po9ycynwq8yxdgg2qwq9e9qpyrtre",
+            "agreement": "jkdjsvd8f78ffkwfek2q"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/2po9ycynwq8yxdgg2qwq9e9qpyrtre/transactions/view",
+        "query": {
+          "agreement": [
+            "jkdjsvd8f78ffkwfek2q"
+          ]
+        }
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "total": 3,
+          "count": 3,
+          "page": 1,
+          "results": [
+            {
+              "amount": 1234,
+              "transaction_id": "askd87sfnsadn489nfd8ddnv",
+              "name": "J. Citizen",
+              "email": "j.citizen@mail.fake",
```

Comment:
I see we already use `.fake` in our code, we updated some of it to use `*example.com` or `*.test`, so may be it's good to use test tlds epecially for emails?